src/components/Navbar/Navbar.jsx

```diff
@@ -57,8 +58,12 @@ const Navbar = () => {
           <NavLink to="/wishlist" className="btn icon-btn link">
             <i className="far fa-heart"></i>
           </NavLink>
-          <NavLink to="/cart" className="btn icon-btn link">
-            <i className="fas fa-shopping-cart"></i>
+          <NavLink
+            to="/cart"
+            className="btn icon-btn link badge-container icon-container"
+          >
+            <i className="fas fa-shopping-cart "></i>
+            {encodedToken && <span class="badge">{cartProducts.length}</span>}
```

Comment:
Suggestion: You could use Navlink className isActive function to style the currently selected page.

For reference check: https://ultimatecourses.com/blog/active-navlink-classes-with-react-router

Reply:
Thank you Puneet